pkg/cache/keyfunc.go

```diff
@@ -66,7 +67,7 @@ func ToClusterAwareKey(cluster, namespace, name string) string {
 
 // SplitMetaClusterNamespaceKey returns the namespace and name that
 // MetaClusterNamespaceKeyFunc encoded into key.
-func SplitMetaClusterNamespaceKey(key string) (clusterName logicalcluster.Name, namespace, name string, err error) {
+func SplitMetaClusterNamespaceKey(key string) (cluster logicalcluster.Name, namespace, name string, err error) {
```

Comment:
revert? also all the others

Reply:
reverted to `clusterName`, just for the record, we will use:
`clusterName` for `logicalcluster.Name` and `clusterPath` for `logicalcluster.Path`